cves/CVE-2014-1418.yml

```diff
@@ -274,7 +309,15 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This vulnerability was one that was obscure and difficult to notice, but
+    caused a pretty big potential issue. It went unnoticed for nearly 6 years,
```

Comment:
"but caused a pretty big potential issue", this is a bit awkward phrasing. maybe try "but could cause a big issue"